spec/services/tempfile_service_spec.rb

```diff
@@ -21,8 +23,7 @@ def has_content?
 
   let(:file) { class_with_metadata_extraction.new(initialization_options) }
 
-  subject { described_class.new(file) }
-  context '#tempfile' do
+  context 'with a tempfile' do
```

Comment:
Not `describe '#tempfile' do`?

Reply:
Yeah, rubocop is complaining that a `context` should always start with when/with/etc... I'm happy to exclude if we feel like it.

Reply:
@mjgiarlo you're right, updated.